android/app/src/main/java/com/reactnativenavigation/activities/BottomTabActivity.java

```diff
@@ -65,6 +72,20 @@ public void run() {
         });
     }
 
+    protected void setupDrawer(Drawer drawer, Screen screen) {
```

Comment:
I don't see why not,  I was just keeping in line with the setup functions being defined in the derived classes.